SampleGallery/Samples/SDK 10586/CustomConnectedAnimation/ConnectedAnimationShell.xaml.cs

```diff
@@ -29,7 +29,7 @@ public ConnectedAnimationShell()
         {
             InitializeComponent();
 
-            SampleComboBox.ItemsSource = new[] { "XAML Connected Animation", "Custom Connected Animation" };
```

Comment:
Was this API superseded or just not brought forward yet?

Reply:
SystemNavigationManager is UWP only and isn't part of WinAppSDK. The newer mechanism "back button" is WinUI only, so AFAIK, this "back button without WinUI" scenario doesn't exist.